src/string-class.js

```diff
@@ -0,0 +1,137 @@
+' use strict';
+
+const StringClass = {
+  /**
+   * Tests if a string contains vowels
+   * @return {Boolean} True or false depending on the match
+   */
+  hasVowels() {
+    return /[aeiouAEIOU]/g.test(this);
+  },
+  /**
+   * converts strings to uppercase
+   * @return {String} string that has been converted to uppercase
+   */
+  toUpper() {
+    return this.replace(/[a-z]/g, (character) => {
+      return String.fromCharCode(character.charCodeAt() - 32);
+    });
+  },
+  /**
+   * converts strings to lowercase
+   * @return {String} string that has been converted to lowercase
+   */
+  toLower() {
+    return this.replace(/[A-Z]/g, (character) => {
+      return String.fromCharCode(character.charCodeAt() + 32);
+    });
+  },
+  /**
+   * converts strings to title case
+   * @return {String} string that has been converted to title case
+   */
+  ucFirst() {
+    return `${this.slice(0, 1).toUpper()}${this.slice(1).toLower()}`;
+  },
+  /**
+   * checks if a string is a question by looking for '?' sign
+   * @return {Boolean} true or false depending on the match
+   */
+  isQuestion() {
+    return /^[A-Za-z0-9\s]+\?$/.test(this.trim());
+  },
+ /**
+   * checks for words in a string
+   * @return {String} returns array of words in the string
+   */
+  word() {
+    return this.replace(/[^\w\s]/g, '').split(/\s+/);
+  },
+  /**
+   * checks the number of words present in a string
+   * @return {Number} the number of words in the string
+   */
+  wordCount() {
+    return this.word().length;
+  },
+  /**
+   * converts currency representation to string
+   * @return {String} currency representation of the the number in String
+   */
+  toCurrency() {
+    let roundCurrency = String(Math.round(((Number(this)) + 0.00001) * 100) / 100);
```

Comment:
This looks a bit complicated, you should probably check out `.toFixed()`. [Check here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed)